tests/test_siwe.py

```diff
@@ -2,19 +2,20 @@
 import json
 from humps import decamelize
 from eth_account import Account, messages
+from dateutil.parser import isoparse
 
 
-from siwe.siwe import SiweMessage, ValidationError
+from siwe.siwe import SiweMessage, VerificationError
 
 BASE_TESTS = "tests/siwe/test/"
 with open(BASE_TESTS + "parsing_positive.json", "r") as f:
     parsing_positive = decamelize(json.load(fp=f))
 with open(BASE_TESTS + "parsing_negative.json", "r") as f:
     parsing_negative = decamelize(json.load(fp=f))
-with open(BASE_TESTS + "validation_negative.json", "r") as f:
-    validation_negative = decamelize(json.load(fp=f))
-with open(BASE_TESTS + "validation_positive.json", "r") as f:
-    validation_positive = decamelize(json.load(fp=f))
+with open(BASE_TESTS + "verification_negative.json", "r") as f:
+    verification_negative = decamelize(json.load(fp=f))
+with open(BASE_TESTS + "verification_positive.json", "r") as f:
+    verification_positive = decamelize(json.load(fp=f))
 
 
 class TestMessageParsing:
```

Comment:
I just noticed, on line 29 there's a comparison but without an `assert` so it's useless, could you add it please?